script/windows-installer/inno-setup-git-lfs-installer.iss

```diff
@@ -15,7 +17,7 @@
 AppId={{286391DE-F778-44EA-9375-1B21AAA04FF0}
 AppName={#MyAppName}
 AppVersion={#MyAppVersion}
-;AppVerName={#MyAppName} {#MyAppVersion}
+AppCopyright=GitHub, Inc. and Git LFS contributors
```

Comment:
I think this is 🆗 since it matches the text in `LICENSE.md`, but I would love to get some extra :eyes: on this just to be sure.

cc @bkeepers